rrule/rrule.d.ts

```diff
@@ -158,4 +158,27 @@ declare namespace RRule {
 
     function fromString(value: string): RRule;
 
+    class RRuleSet extends RRule {
```

Comment:
yeah, it's not on 2.1.0. But it's a very handy feature that has been on master for 9 months at least. If it's really strict to be present on npm release, I can send another diff with just the other changes (`between` and `all`), otherwise this might help people like me who decide to go with the latest version on git.